pkg/serverconfig/types.go

```diff
@@ -129,3 +129,9 @@ type HelmChartRepo struct {
 type Helm struct {
 	ChartRepo HelmChartRepo `yaml:"chartRepository"`
 }
+
+type ManagedClusterConfig struct {
+	Name   string `json:"name" yaml:"name"`
+	Server string `json:"server" yaml:"server"`
```

Comment:
I would be explicit and say `apiServerURL` so it's clear it's not just a hostname